dev/AppNotifications/AppNotifications.idl

```diff
@@ -111,6 +111,29 @@ namespace Microsoft.Windows.AppNotifications
 
         // Gets or sets whether a Notification's pop-up UI is displayed on the user's screen.
         Boolean SuppressDisplay;
+
+        // Gets or sets the Notification Device Data
+        AppNotificationDevicesData DevicesData;
```

Comment:
This should probably be renamed so it's clearly about Calling - it's not general purpose "here is device information about this notification."  Some potential names for this type:

* AppNotificationConferencingConfig

(Then change the name of this member to match, like `ConferencingConfig`)

Reply:
@jonwis , Sure make sense, will rename to AppNotificationConferencingConfig

Reply:
Have renamed to AppNotificationConferencingConfig , except the classes those are implementing UDK interfaces, its left as DevicesData, hope it fine.
@anupriya13

Reply:
I will update in FrUDK too soon